mycore-indexing/src/main/java/org/mycore/frontend/indexbrowser/MCRGoogleSitemapCommon.java

```diff
@@ -112,16 +112,16 @@ public final class MCRGoogleSitemapCommon {
         MCRSolrAuthenticationManager.getInstance();
 
     /** The logger */
-    private static Logger LOGGER = LogManager.getLogger(MCRGoogleSitemapCommon.class.getName());
+    private static final Logger LOGGER = LogManager.getLogger(MCRGoogleSitemapCommon.class.getName());
 
     /** Number of URLs in one sitemap */
     private int numberOfURLs = MCRConfiguration2.getInt("MCR.GoogleSitemap.NumberOfURLs").orElse(10000);
 
     /** number format for parts */
-    private static NumberFormat number_format = getNumberFormat();
+    private static final NumberFormat NUMBER_FORMAT = getNumberFormat();
 
     /** date formatter */
-    private static SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd", SITEMAP_LOCALE);
+    private static final SimpleDateFormat DATE_FORMATTER = new SimpleDateFormat("yyyy-MM-dd", SITEMAP_LOCALE);
```

Comment:
Should be replaced with https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/time/format/DateTimeFormatter.html which is thread safe